package.json

```diff
@@ -77,8 +77,8 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.0.3",
     "@types/node": "^14.14.22",
-    "@types/react": "^17.0.1",
-    "@types/react-dom": "^17.0.1",
+    "@types/react": "^16.9.11 || ^17.0.1",
+    "@types/react-dom": "^16.9.3 || ^17.0.1",
```

Comment:
I don't think that we need to support the older React version for devDependencies, just for peer dependencies.

Reply:
switched dev dependencies, "@types/react" and "@types/react-dom" to be 17.0.1 in latest push